scripts/version

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
Do we really need this script? AFAIK this is normally used for the `II_VERSION` build parameter, which we obviously don't use. I think ee only use it to echo its value at some point, so why not removing it entirely?